src/app/components/harvest/pages/list/list.component.ts

```diff
@@ -44,11 +44,62 @@ class ListComponent extends PageComponent implements OnInit {
   public ngOnInit(): void {
     this.project = this.route.snapshot.data[projectKey].model;
     this.canCreateHarvestCapability = this.project.can("createHarvest").can;
+    // A BehaviorSubject is need on fitlers$ to update the ngx-datatable harvest list & models
+    // The this.filters$ is triggered in abortUpload()
+    this.filters$ = new BehaviorSubject({
+      sorting: {
+        direction: "desc",
+        orderBy: "createdAt",
+      }
+    });
+  }
+
+  public async abortUpload(template: any, harvest: Harvest): Promise<void> {
+    const ref = this.modals.open(template);
+    const success = await ref.result.catch((_) => false);
+
+    if (success) {
+      this.harvestsApi
+      .transitionStatus(harvest, "complete")
+      .pipe(
+        catchError((err: BawApiError) => {
+          if (err.status !== CLIENT_TIMEOUT) {
+            return throwError(() => err);
+          }
+        })
+      )
+      // We want this api request to complete regardless of lifecycle destruction
+      // eslint-disable-next-line rxjs-angular/prefer-takeuntil
+      .subscribe({
+        next: (): void => {
+          this.filters$.next({
+            sorting: {
+              direction: "desc",
+              orderBy: "createdAt"
+            }
```

Comment:
do we need to set this value? can we just call next with an empty object?

Reply:
Calling with an empty object does force the filters$ to update
I have since made the necessary code changes